src/CustomElementInternals.js

```diff
@@ -208,6 +222,11 @@ export default class CustomElementInternals {
             this.patchAndUpgradeTree(importNode, visitedImports);
           });
         }
+      } else if (element.hasAttribute('is') || element.is) {
```

Comment:
The `is=""` attribute should only be looked at when an element is being created, but never again. This function is run when `innerHTML` occurs, so it is when elements are newly created. So this is the only place we respect what is in the `is` attribute.